recipes/cds-crtools/meta.yaml

```
@@ -0,0 +1,971 @@
+        then generate filter files usable by the LIGO real-time
+        data acquisition system.
+        A variety of design strategies are supported.
+
+  - name: lidax
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+      host:
+        - {{ pin_subpackage('crtools-gui-libraries', exact=True) }}
+        - gds-base
+        - {{ pin_subpackage('libdfm', exact=True) }}
+        - root_base
+      run:
+        - {{ pin_subpackage('crtools-gui-libraries', exact=True) }}
+        - gds-base
+        - {{ pin_subpackage('libdfm', exact=True) }}
+        - root_base
+    files:
+      - bin/lidax
+      - bin/udnls
+    test:
+      commands:
+        - lidax -h
+        - lidax -H NDSSERVER
+        - udnls -h
+    about:
+      home: https://git.ligo.org/cds/dtt
+      dev_url: https://git.ligo.org/cds/dtt.git
+      license: GPL-2.0-or-later
+      license_family: GPL
+      license_file:
+        - COPYING
+        - COPYING-GPL-3
+      summary: Graphical program for retrieving LIGO data.
+      description: |
+        lidax reads LIGO frame files.
+
+  # -- metapackage ----------
+
+  - name: cds-crtools
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        # all of the outputs
+        - {{ pin_subpackage('chndump', exact=True) }}
+        - {{ pin_subpackage('dmtviewer', exact=True) }}
+        - {{ pin_subpackage('dtt-diag', exact=True) }}
+        - {{ pin_subpackage('dtt-diagd', exact=True) }}
+        - {{ pin_subpackage('dtt-diaggui', exact=True) }}
+        - {{ pin_subpackage('dtt-awggui', exact=True) }}
+        - {{ pin_subpackage('dtt-awgstream', exact=True) }}
+        - {{ pin_subpackage('dtt-multiawgstream', exact=True) }}
+        - {{ pin_subpackage('dtt-tpcmd', exact=True) }}
+        - {{ pin_subpackage('dtt-xml-tools', exact=True) }}
+        - {{ pin_subpackage('foton', exact=True) }}
+        - {{ pin_subpackage('lidax', exact=True) }}
+        - {{ pin_subpackage('python-awg', exact=True) }}
+        - {{ pin_subpackage('python-foton', exact=True) }}
+    test:
+      commands:
+        - diaggui -h
+        - foton -h
+    about:
+      home: https://git.ligo.org/cds/dtt
+      dev_url: https://git.ligo.org/cds/dtt.git
+      license: GPL-2.0-or-later
+      license_family: GPL
+      license_file:
+        - COPYING
+        - COPYING-GPL-3
+      summary: CDS Control Room tools
+      description: |
+        cds-crtools is a metapackage that installs several programs
+        used to control LIGO-style interfrometers. The package includes
+        `diag`, `diagd`, `diaggui`, `awggui`, `tpcmd`, `chndump`
+        along with `foton`, the filter design program, and others.
+
+about:
+  home: https://git.ligo.org/cds/dtt
+  dev_url: https://git.ligo.org/cds/dtt.git
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file:
+    - COPYING
+    - COPYING-GPL-3
+  summary: CDS Control Room tools
+
+extra:
+  recipe-maintainers:
```

You can control the feedstock name to avoid having `-split` in it:
```suggestion
  feedstock-name: cds-crtools
  recipe-maintainers:
```